svf/lib/MSSA/SVFGBuilder.cpp

```diff
@@ -41,15 +41,18 @@ using namespace SVFUtil;
 
 SVFG* SVFGBuilder::buildPTROnlySVFG(BVDataPTAImpl* pta)
 {
-    if(Options::OPTSVFG())
+    if(Options::OPTSVFG() || this->OptimiseSVFG)
```

Comment:
remove `Options::OPTSVFG()` if you change line 5 in file [‎svf/include/MSSA/SVFGBuilder.h](https://github.com/SVF-tools/SVF/commit/def7c043fe32b6bee0b94c68d2de95b3f0b7d2df#diff-85ba8e86274b5c170911a557b57579558efb1f00bea88eb44410826f724d153f)